floyd/cli/experiment.py

```diff
@@ -164,26 +165,27 @@ def delete(ids, yes):
     """
     Delete project runs
     """
-    
+    failures = False
     for id in ids:
         experiment = ExperimentClient().get(id)
-        task_instance_id = get_module_task_instance_id(experiment.task_instances)
-        task_instance = TaskInstanceClient().get(task_instance_id) if task_instance_id else None
-
+        if not experiment:
+            failures = True
+            continue
 
-        if experiment.state in ["queued", "running"]:
-            floyd_logger.info("Experiment {}: In {} state and cannot be deleted. Stop it first".format(experiment.name, experiment.state))
+        if not yes and not click.confirm("Delete Run: {}?".format(experiment.name),
+                                         abort=False,
+                                         default=False):
+            floyd_logger.info("Experiment {}: Skipped.".format(experiment.name))
             continue
 
-        if not yes:
-            if not click.confirm("Delete Run: {}?".format(experiment.name), abort=False, default=False):
-                floyd_logger.info("Experiment {}: Skipped.".format(experiment.name))
-                continue
+        task_instance_id = get_module_task_instance_id(experiment.task_instances)
+        task_instance = TaskInstanceClient().get(task_instance_id) if task_instance_id else None
 
         if task_instance and task_instance.module_id:
             ModuleClient().delete(task_instance.module_id)
```

Comment:
It seems like ModuleClient.delete could fail but the delete operations will continue. It should be added to the failures at least so that the user can take appropriate action to delete it later.